Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift

```diff
@@ -558,7 +570,7 @@ public final class PackageInfoMapper: PackageInfoMapping {
                         return nil
                     }
                     if let target = packageInfo.targets.first(where: { $0.name == name }) {
-                        if target.type == .binary, case let .external(artifactPaths: artifactPaths) = packageType {
+                        if target.type == .binary, case let .remote(artifactPaths: artifactPaths) = packageType {
```

Comment:
Can a local package be binary? I assume the answer is no, but would love to get a confirmation.

Reply:
If a local package manifest is configured like a structure below, a local target can be binary as a dependency of some targets.

```swift
let package = Package(
    name: "MyLocalPackage",
    targets: [
        .binaryTarget(name: "LocalBinary", path: "artifacts/local-binary.zip"),
        .target(name: "MyLocalTarget", dependencies: [.target(name: "LocalBinary")]),
    ]
)
```

Can you let me know why you are curious?

Reply:
Doesn't `case let .remote(artifactPaths: artifactPaths) = packageType` mean we would fallback to returning a regular target instead of an `.xcframework` if we have a local package binary?

Can we test this scenario with a unit test? We should test the following:
- Given a local binary package binary dependency
- I expect the binary target to be integrated as an `.xcframework`

Reply:
It is an amazing catch! 👍 I didn't expect to affect this point. Actually, 'target can't found' error was raised for a binary target of a local package.
Then I've added a condition for a binary target of a local package and relevant tests.

Reply:
@woin2ee just an FYI that #6676 has fixed the issue of targets not being found when a local package references a binary .zip. Not sure if that impacts your test cases but figured I'd mention it!